http/parallel_file_server.ts

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env -S deno serve --allow-read
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+/**
+ * This module is a CLI for serving static files on multiple threads.
+ *
+ * @example Start the server
+ * ```sh
+ * deno serve --allow-read --parallel jsr:@std/http/parallel-file-server
+ * ```
+ *
+ * @example Get help with `deno serve`
+ *
+ * By using the `--help` flag.
+ *
+ * ```sh
+ * deno serve --help
+ * ```
+ *
+ * @example Define port and hostname to listen on
+ *
+ * By setting the `--port` and `--hostname` flags.
+ *
+ * ```sh
+ * deno serve --allow-read --parallel --port=8080 --hostname=1.2.3.4 jsr:@std/http/parallel-file-server
+ * ```
+ *
+ * @example Enable TLS
+ *
+ * By setting the `--cert` and `--key` flags.
+ *
+ * ```sh
+ * deno serve --allow-read --parallel --cert=cert.pem --key=key.pem jsr:@std/http/parallel-file-server
+ * ```
+ *
+ * @example Disable directory listing
+ *
+ * By setting the `--no-dir-listing` flag.
+ *
+ * ```sh
+ * deno serve --allow-read --parallel --no-dir-listing jsr:@std/http/parallel-file-server
```

Comment:
`--no-dir-listing` needs to be after the script name
```suggestion
 * deno serve --allow-read --parallel jsr:@std/http/parallel-file-server --no-dir-listing
```